orcoursetrion/actions/github.py

```diff
@@ -38,3 +38,45 @@ def create_export_repo(course, term, description=None):
         config.ORC_STUDIO_ORG, repo_name, config.ORC_STUDIO_DEPLOY_TEAM
     )
     return repo
+
+
+def create_xml_repo(course, term, team, description=None):
```

Comment:
For now that would be the case, but based on the design this action is going to diverge more and more over time.  It already has two differences, the hook and the extra team, and when we add a course skeleton it will diverge more, so I think I'll keep them separate.  I went back and forth on it a lot.